### ISSUE TITLE
KPMP tissue blocks not rendered in Body UI, only shown in details pane

### ISSUE THREAD

**Issue by andreasbueckle** [hubmapconsortium/ccf-ui#938]
When filtering for KPMP as Tissue Provider: 

![image](https://user-images.githubusercontent.com/22821046/184952508-a45ee1ef-a225-42fd-926d-35724b72ec8d.png)

**Comment by andreasbueckle** [comment]
@bherr2 

Is it because this `target ` URL is outdated? 
https://github.com/hubmapconsortium/ccf-ui/blob/bd96d1a237189a20f90240e4b8b0b66be97eb98e/projects/ccf-eui/src/assets/kpmp/data/rui_locations.jsonld#L45

**Comment by bherr2** [comment]
Yep! Those are OLD placements. I have a patch for those, but I need to know whether these should be Male or Female. @andreasbueckle can you track this bit down? I can apply the Patch after I know the sex.

**Comment by andreasbueckle** [comment]
@bherr2 
I dug up these notes from Seth Winfree (the creator) from May 18, 2020. We only had the male left kidney available at the time. Surprised the target `URL` is `http://purl.org/ccf/latest/ccf.owl#VHRightKidney`. So But for sex, it must be male. 

-------------------------
Date: 05/18/2020

Author:	Seth Winfree



Objective: Use RUI to position Biop8,7 and JASN Cover nephrectomy volume.  RUI are "best guesses". 



Biop8

aliases: Myeloperoxidase_CD45R_Biops8_2_RGB.tif

RUI_xml:{

    "alignment_id": "59411967-28d2-4955-833e-7f8a8ab75d85",

    "alignment_operator_first_name": "Seth",

    "alignment_operator_last_name": "Winfree",

    "alignment_datetime": "5/18/2020 1:25:58 PM",

    "tissue_position_mass_point": {

        "x": 7.0,

        "y": 63.0,

        "z": 12.0

    },

    "tissue_object_rotation": {

        "x": 1.6699999570846558,

        "y": 11.710000038146973,

        "z": 261.29998779296877

    },

    "tissue_object_size": {

        "x": 1.08336,

        "y": 4.06097,

        "z": 0.11280000149011612

    }

}

Notes: Biopsy from Nephrology biobank-salvaged from pathology in ~2017.  

Imagesize: 1002x3756x107 pixels

Volume:	1083.36x4060.97x111.28 microns 

Channels: 4

Channel description: {"DAPI", "Phalloidin", "Myeloperoxidase", "CD45R"}

Channel LUT: {"White/Gray", 'Green", "Red", "Blue"} 

Reagents:  As given in IU/OSU KPMP MOP

Protocol DOI: dx.doi.org/10.17504/protocols.io.9avh2e6

Published DOI: dx.doi.org/10.1681/ASN.2016091027

Comment: Cortical biopsy



Biop7

aliases: Myeloperoxidase_CD45R_Biops7_1_RGB.tif

RUI_xml: {

    "alignment_id": "c7362760-04fd-4679-a694-b25592ef0291",

    "alignment_operator_first_name": "Seth",

    "alignment_operator_last_name": "Winfree",

    "alignment_datetime": "5/18/2020 1:30:26 PM",

    "tissue_position_mass_point": {

        "x": 6.0,

        "y": 66.0,

        "z": 19.0

    },

    "tissue_object_rotation": {

        "x": 359.6700134277344,

        "y": 0.0,

        "z": 278.0299987792969

    },

    "tissue_object_size": {

        "x": 1.2001300476837159,

        "y": 5.423270095367432,

        "z": 0.07696000149011612

    }

}

Notes: Biopsy from Nephrology biobank-salvaged from pathology in ~2017.

Imagesize: 1110x5016x74 pixels

Volume:	1200.13x5423.27x76.96 microns 

Channels: 4

Channel description: {"DAPI", "Phalloidin", "Myeloperoxidase", "CD45R"}

Channel LUT: {"White/Gray", 'Green", "Red", "Blue"} 

Reagents:  As given in IU/OSU KPMP MOP

Protocol DOI: dx.doi.org/10.17504/protocols.io.9avh2e6

Published DOI: dx.doi.org/10.1681/ASN.2016091027

Comment: Cortical biopsy



CoverNephrectomy

aliases: Fused_8bit-1.tif

RUI_xml: {

    "alignment_id": "bd7a9bea-a726-427a-8360-8a017ef8178d",

    "alignment_operator_first_name": "Seth",

    "alignment_operator_last_name": "Winfree",

    "alignment_datetime": "5/18/2020 1:34:07 PM",

    "tissue_position_mass_point": {

        "x": 10.0,

        "y": 67.0,

        "z": 19.0

    },

    "tissue_object_rotation": {

        "x": 359.6700134277344,

        "y": 3.3499999046325685,

        "z": 10.039999961853028

    },

    "tissue_object_size": {

        "x": 3.44736,

        "y": 7.776000190734863,

        "z": 0.07384000149011612

    }

}

Note: Isolated as a part of a kidney stone PPG.

Imagesize:  3192x7200x71 pixels

Volume:	 3447.36x7776.00x73.84 microns 

Channels: 6-7

Channel description: {"Phalloidin bandpass1", "Phalloidin bandpass2", "Uromodulin", "Myeloperoxidase", "Aquaporin-1", "CD68", "DAPI"}

Channel LUT: {"Green", 'Green", "Cyan", "Red", "Magenta", "Yellow", "White"} 

Reagents:  As given in IU/OSU, KPMP MOP, Staining panel#1, fluorescein and OG488 approach.

Protocol DOI: similar to dx.doi.org/10.17504/protocols.io.9avh2e6

Published DOI: dx.doi.org/10.1016/j.trsl.2017.07.006

Published URL: jasn.asnjournals.org/content/28/7.cover-expansion

Comments: Volume includes Medula(outer stripe and inner stripe) and Cortex with clear arcuate vessels.

**Comment by bherr2** [comment]
Ok, do you want to re-place those in the left kidney (I don't think I have patches to do that automatically) or keep it in the right kidney?

**Comment by andreasbueckle** [comment]
Let's try the right kidney first. Maybe via a preview build. Doable? If it fits, we can ask Seth to confirm.

**Comment by bherr2** [comment]
Good idea. Will do that shortly.

**Comment by bherr2** [comment]
And actually it looks like I can just change the patch from Right to Left and it'll work. But maybe verify with Seth first.

**Comment by andreasbueckle** [comment]
Sounds good. Looking forward to the preview!

**Comment by bherr2** [comment]
Fixed in production
